consensus/src/tests/scenarios_reward_split.rs

```diff
@@ -220,7 +220,7 @@ async fn test_reward_split() {
                         .unwrap()
                         .checked_div_u64((1 + cfg.endorsement_count).into())
                         .unwrap()
-                        .saturating_sub(third.checked_mul_u64(2).unwrap())
```

Comment:
That should not change just moving signatures around

Reply:
> That should not change just moving signatures around

You're right, it should be done in another PR. NB: Before it was "2 * 0", I don't understand why we can't see this version of the main branch